source/blog/2016-12-14-size-and-change.html.md

```diff
@@ -0,0 +1,165 @@
+they met on their way to their desk, and the people in their immediate
+office.
+
+* As mailing lists grow, the peer pressure to respond personally to
+any individual message diminishes. At a certain size, it becomes much
+more difficult to evaluate the authority level of a person posting on
+the list, and the social ties weaken between list participants. On one
+list, when the main list got too busy, they created a "developers"
+mailing list where they could hang out together. A similar phenomenon
+happens with IRC channels.
+
+* I remember talking to a friend who was the founder of a growing
+tech company, who told me about the first time his company hired someone
+he had not personally interviewed&mdash;my recollection is that it happened
+after the company passed 50 employees, but before it hit 70. Around the
+time the company had 50 people, the overhead of interviewing and hiring
+new people grew beyond his ability to scale.
+
+There are multiple other scaling events&mdash;when do you implement some
+kind of HR process for absence management, expenses, pay raises, and so
+on? How big are you when it becomes cost effective to hire a full time
+accountant, lawyer, administrator, talent acquisition? When is a single
+team in your company big enough that you need to hire or promote someone
+who is not a founder to management? How big before it is unreasonable to
+ask everyone's opinion on a work change that impacts everyone?
+
+These events are specific to companies, but communities have similar
+scaling experiences. At some point, the project maintainer will not be
+personally reviewing all changes to the project. Tools and process will
+need to be added for managing bug reports, releases, automated builds,
+source control. As the project gets bigger, you will hit a point where
+it is more work than volunteers can do to maintain infrastructure, and
+your project may need to budget money for a sysadmin. And if you need
+budget for a sysadmin, you may be at a point whether it is worthwhile
+having someone work on fundraising, business development, content
+management, and other tasks that community volunteers traditionally do
+not do well.
+
+## Structure and Relationships
+
+At each of these changes, required structure and process is one axis.
+The other axis, perhaps more significant, is how people in the group
+relate to each other. Individuals can have multiple identities at once,
+each of which is stronger or weaker. I can be part of a project and part
+of a company at the same time&mdash;and whether I consider myself an project
+foo developer who gets paid by company X, or a company X employee who
+happens to be working on project foo, is a very personal thing,
+depending on the depth of the relationships that I have with other
+members of the groups "project foo" and "company X."
+
+The relationships with group members will change over time, as group
+size changes. I started my developer career as a developer of the GIMP,
```

Comment:
Maybe "open source developer career"? It's more accurate, but I'll let the editor decide if it makes the prose too cumbersome ;-)